lab_3_generate_by_ngrams/main.py

```diff
@@ -76,6 +98,12 @@ def get_token(self, element_id: int) -> Optional[str]:
 
         In case of corrupt input arguments or arguments not included in storage, None is returned
         """
+        if not isinstance(element_id, int) or element_id not in self._storage.values():
+            return None
+
+        for key in self._storage:
```

Comment:
try to use `.items()` to avoid using the `[]` operator

Reply:
done